### PR TITLE
GCE: neg to use generated code

### PR DESCRIPTION
**Pull request by bowei**
GCE: neg to use generated code

```release-note
NONE
```

### CODE REVIEW

File: pkg/cloudprovider/providers/gce/gce_networkendpointgroup.go [hunk 1]
```diff
@@ -32,117 +37,96 @@ func newNetworkEndpointGroupMetricContext(request string, zone string) *metricCo
...
-			}
-			zoneNetworkEndpointGroupMap[zone] = append(zoneNetworkEndpointGroupMap[zone], negs.NetworkEndpointGroups...)
+	// TODO: filter for the region the cluster is in.
+	all, err := gce.c.AlphaNetworkEndpointGroups().AggregatedList(context.Background(), filter.None)
+	if err != nil {
+		return nil, mc.Observe(err)
+	}
+	ret := map[string][]*computealpha.NetworkEndpointGroup{}
+	for key, byZone := range all {
+		parts := strings.Split(key, "/")
```
Comment (freehan) [inline pkg/cloudprovider/providers/gce/gce_networkendpointgroup.go]: add the comment: "// key has the format of "zones/${zone_name}"

**Comment by freehan** [comment]
Only nits.

**Comment by bowei** [comment]
done

**Pull request merged**